src/chttpd_handlers.erl

```diff
@@ -51,6 +54,20 @@ design_handler(HandlerKey, DefaultFun) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+is_disabled(KeyBin) ->
+    BlacklistConfig = config:get("chttpd", "url_handler_blacklist", []),
```

Comment:
I played a bit with your proposal. One more point against lists: it's hard to operate with them via API. 
You cannot simple remove one endpoint from blacklist: you need to GET `/_config/chttpd/httpd_global_handler_blacklist`, decode, pop element, encode, PUT it back. Same story if you need to add something to blacklist. Compare it with a single PUT/DELETE `/_config/chttpd_global_handlers_blacklist/_uuids` request, which both are atomic.

Simplicity of using that blacklist via API also matters, imho.